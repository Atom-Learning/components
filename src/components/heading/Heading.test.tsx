import { render, screen } from '@testing-library/react'
import { axe } from 'jest-axe'
import * as React from 'react'

import { Heading } from '.'

describe(`Heading component`, () => {
  it('renders a heading', async () => {
    const { container } = render(<Heading>HEADING</Heading>)

    await screen.getByText('HEADING')

    expect(container).toMatchSnapshot()
    expect(await axe(container)).toHaveNoViolations()
  })
})
