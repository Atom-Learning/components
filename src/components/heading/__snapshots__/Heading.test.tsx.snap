// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component renders a heading 1`] = `
.sx53fkz {
  color: var(--sx-colors-tonal900);
  font-family: var(--sx-fonts-display);
  font-weight: 700;
  margin: 0;
}

.sx53fkzq3glm--size-md {
  font-size: var(--sx-fontSizes-xl);
  line-height: 1.14;
}

.sx53fkzq3glm--size-md::before {
  content: '';
  margin-bottom: -0.2174em;
  display: table;
}

.sx53fkzq3glm--size-md::after {
  content: '';
  margin-top: -0.2254em;
  display: table;
}

<div>
  <h2
    class="sx53fkz sx53fkzq3glm--size-md"
  >
    HEADING
  </h2>
</div>
`;
